homeassistant/components/control4/strings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "The Control4 controller associated with this account has already been configured."
+    },
+    "error": {
+      "cannot_connect": "Couldn't connect to the local controller. Check your hostname and login details.",
+      "invalid_auth": "Invalid Control4 username or password.",
+      "unknown": "An unknown error occured. Please report log info."
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "Controller hostname",
+          "password": "Password",
+          "username": "Username/Email"
+        },
+        "title": "Control4",
```

Comment:
This is not necessary. If you omit it, it's automatically using the integration name.
```suggestion
```